libbeat/common/_meta/test/json_map_dedot.json

```diff
@@ -3,8 +3,8 @@
         "key_with_dot_l2": 1,
         "key_with_multiple_dots_l2": 2,
         "key_without_dot_l2": {
-            "key_with_dot_l2": 3,
-            "key_with_multiple_dots_l2": 4
+            "key_with_dot_l3": 3,
```

Comment:
Let's call the directory `testdata` which is more idiomatic for Golang. The `_meta` data directory is specific to our modules where we automatically generate some information, we don't need it here I think.

Reply:
Moved the test data to testdata folder